crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,26 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def check_unclaimed_disks
```

Comment:
It think the name is a bit missleading. We're not trying to find unclaimed disks. But we're try to check whether the devices in the existing claims to still exist.

How about calling it verify_disk_claims or something like that.